lace/test_obj.py

```diff
@@ -81,6 +82,18 @@ def test_loading_brings_in_normals_and_uvs(self):
         self.assertEqual(mesh_with_texture.vn.shape[1], 3)
         self.assertEqual(mesh_with_texture.vn.shape[0], np.max(mesh_with_texture.fn)+1)
 
+    def test_loading_vertex_colors(self):
+        # Mesh without vertex colors should not have vertex colors
+        mesh_without_vertex_colors = obj.load(sc(self.test_obj_url))
+        self.assertIsNone(mesh_without_vertex_colors.vc)
+
+        # Mesh with vertex colors should have vertex colors
+        mesh_with_vertex_colors = obj.load(sc(self.test_obj_with_vertex_colors_url))
+        self.assertIsNotNone(mesh_with_vertex_colors.vc)
+
```

Comment:
Maybe assert the shape of the `vc` array is correct.
Or even more strictly that `vc` is `np.allclose` to the expected

```
array([[ 0. ,  0. ,  0. ],
       [ 0.5,  0. ,  0. ],
       [ 1. ,  0. ,  0. ],
       [ 0.5,  0.5,  0. ],
       [ 0. ,  1. ,  0. ],
       [ 0.5,  0.5,  0.5],
       [ 0. ,  0. ,  1. ],
       [ 1. ,  1. ,  1. ]])
```